packages/babel-standalone/README.md

```diff
@@ -16,11 +16,11 @@ It's true that using Babel through Webpack, Browserify or Gulp should be suffici
 Installation
 ============
 
-There are several ways to get a copy of babel-standalone. Pick whichever one you like:
+There are several ways to get a copy of @babel/standalone. Pick whichever one you like:
 
-- Use it via UNPKG: https://unpkg.com/babel-standalone@6/babel.min.js. This is a simple way to embed it on a webpage without having to do any other setup.
-- Install via Bower: `bower install babel-standalone`
-- Install via NPM: `npm install --save babel-standalone`
+- Use it via UNPKG: https://unpkg.com/@babel/standalone@6/babel.min.js. This is a simple way to embed it on a webpage without having to do any other setup.
+- Install via Bower: `bower install @babel/standalone`
```

Comment:
This is definitely not right, but maybe we should just remove this line now :P

Reply:
I'm +1 to drop Bower